website/auth.py

```diff
@@ -120,8 +121,10 @@ def forget_current_user():
 
 
 def is_admin(user):
-    admin_user = os.getenv('ADMIN_USER')
-    return user.get('username') == admin_user or user.get('email') == admin_user
+    # Get the value from the environment, use literal_eval to convert from string list to an actual list
+    admin_users = os.getenv('ADMIN_USERS')
+    admin_users = ast.literal_eval(admin_users)
```

Comment:
Great job! If I may offer a suggestion, I would try to support both methods of specifying an admin user at the same time.

If you don't, logging in will be impossible during the period that code and config don't match up. Either:

* Deploy code first,  logging in stops working, update config
* Update config first, logging in stops working, deploy code.
 
If we support both methods of specifying the admin user, there's a natural progression:

* Deploy code first, logging in still works
* Update config, loggin in still works

Reply:
Agree, will look into this!